src/common/src/config.rs

```diff
@@ -365,7 +365,7 @@ mod default {
     }
 
     pub fn barrier_interval_ms() -> u32 {
-        250
+        1000
```

Comment:
Should we note somewhere (in `toml`?) that the default value without configuration file here differs from the default value in `risingwave.toml`? 🥵